### ISSUE TITLE
turn .json to .txt which will use on darknet

### ISSUE THREAD

**Issue by ujsyehao** [pdollar/coco#31]
Hi, i don't know how to turn the .json file into .txt file .I just want to get the .txt file which stores class number,x ratio,y ratio,width, height. I use the .txt file to train on darknet.Who can help me?

**Comment by ujsyehao** [comment]
@hengck23 No,when i use it ,it have problems ImportError: No module named baker

**Comment by ujsyehao** [comment]
OK,i have written a script which can turn json file into txt file that darknet nedd.
Thank you everyone!

**Comment by Cuijie12358** [comment]
@ujsyehao Hi,sorry to disturb you but can you tell me how to write this scrpit ?
I use python to load from json and write into txt but it didn't work...

**Comment by Cuijie12358** [comment]
Thank you so much!@ujsyehao

**Comment by jibsamGarcia** [comment]
@ujsyehao brother, 

Hi, i don't know how to turn the .json file into .txt file .I just want to get the
--help me...

**Comment by Penelope1rose** [comment]
i cannot even access the link, please help. as for me, i need a txt to .json :( idk where else to go

**Comment by ujsyehao** [comment]
Please look into [it](https://github.com/ujsyehao/COCO-annotations-darknet-format)
